static/js/controllers/contacts-content.js

```diff
@@ -64,6 +65,15 @@ angular.module('inboxControllers').controller('ContactsContentCtrl',
             'ContactsContentCtrl',
             function(areTasksEnabled, tasks) {
               if ($scope.selected) {
+                tasks.forEach(function(task) {
+                  if (_.isString(task.title)) {
+                    // new translation key style
+                    task.title = $translate.instant(task.title, task);
+                  } else {
+                    // old message array style
+                    task.title = TranslateFrom(task.title, task);
```

Comment:
1. is there any difference between `_.isString(x)` vs `typeof x === 'string'`?
2. is this switch something we do regularly?  Should it be handled by a service?

Reply:
1. I don't think so, but I prefer to use underscore unless we're already using it in this module.
2. Yeah good point. It affects a lot of other code so I've created #4083 to work on it.